test-resources/bin/lib/util

```diff
@@ -388,13 +388,27 @@ push_all_images() {
 }
 
 deploy_filelog_offsets_pvc() {
-  echo "STEP $step_counter: install filelog offset persistent volume claim"
-  if [[ "$is_kind_cluster" = "true" ]]; then
```

Comment:
We actually had one last instance of `$is_kind_cluster` here, which we missed in commit 35b9d1376f4e508ebf424c6b58c064db10603fe8. It would crash the script due to the unbound variable.

Since FILELOG_OFFSETS_PVC supports different values (including `kind`, `docker`, etc.) now instead of only true vs. false, this was rather easy to resolve.